pxr/base/plug/testPlug.py

```diff
@@ -275,10 +275,12 @@ def test_ErrorCases(self):
         self.assertEqual(listener.numReceived, 0)
 
         # try to load an unloadable plugin
-        badPlugin = Plug.Registry().GetPluginForType('TestPlugUnloadable')
-        self.assertIsNotNone(badPlugin)
-        with self.assertRaises(RuntimeError):
-            badPlugin.Load()
+        # NOTE: This test is broken on Windows ARM64, so is switched off
```

Comment:
```suggestion
        # XXX: This test is broken on Windows ARM64, so is switched off
```

We typically use `XXX:` comments to denote known issues that should be fixed. Also, could you please expand the comment a bit to include more info about the failure?